metrics/prometheus_machine.go

```diff
@@ -192,7 +193,29 @@ func NewPrometheusMachineCollector(i infoProvider, includedMetrics container.Met
 			},
 		}...)
 	}
+	if includedMetrics.Has(container.VmStatMetrics) {
+		c.machineMetrics = append(c.machineMetrics, []machineMetric{{
+			name:        "machine_vmstat",
```

Comment:
please add this metric to documention https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md